frontend/src/main/scala/bloop/Cli.scala

```diff
@@ -23,10 +23,16 @@ object Cli {
       workingDirectory = ngContext.getWorkingDirectory,
     )
     val command = ngContext.getCommand
-    val args =
+    val args = {
       if (command == "bloop.Cli") ngContext.getArgs
       else command +: ngContext.getArgs
-    val cmd = parse(args, nailgunOptions)
+    }
+    val cmd = {
+      // If no command is given to bloop, we'll receive the script's name.
+      if (command == "bloop")
```

Comment:
Why is this necessary?

Reply:
It's not necessary for this PR, I didn't intend to keep this commit here...
The reason is that at the moment, calling `bloop` without any argument will print `bloop: Command not found` which is extraordinarily surprising for people trying bloop for the first time. I'm happy to remove it and put that in another PR (as I initially intended to) if you prefer.

Reply:
> It's not necessary for this PR, I didn't intend to keep this commit here...
The reason is that at the moment, calling bloop without any argument will print bloop: Command not found which is extraordinarily surprising for people trying bloop for the first time. I'm happy to remove it and put that in another PR (as I initially intended to) if you prefer.

Reply:
Does this happen even with our latest changes in nailgun?